.openapi.yaml

```diff
@@ -376,8 +376,9 @@ components:
         id:
           type: string
           format: uuid
-        position:
+        positionID:
```

Comment:
you should use camelcase in all your models either underscore case but not mixed

Reply:
+